src/Logger/QueueLogger.php

```diff
@@ -93,4 +86,47 @@ public function consume($queue, $consumeTag, $callback)
          * Not Implemented for this adapter
          */
     }
+
+    /**
+     * @param  array $config
+     * @throws Exception
+     * @return array
+     */
+    private function prepareConfig($config)
+    {
+        if (!isset($config['logger'])) {
+            throw new Exception('LoggerAdapter can not be loaded.
+                Check config settings.');
+        }
+
+        $configs = [
+            'host' => 'localhost',
+            'port' => 6379,
+            'key'  => 'logstash',
+            'channel' => 'message-queue-php',
+            'path' => '/var/log/message-queue-php.log'
+        ];
+
+        if (isset($config['logger']['host'])) {
```

Comment:
We need throws exceptions if a mandatory parameter is necessary. I dont recommend use default params to connect to a (data) server